parcels/examples/example_globcurrent.py

```diff
@@ -4,64 +4,68 @@
 from glob import glob
 import numpy as np
 import pytest
+import xarray as xr
 
 
 ptype = {'scipy': ScipyParticle, 'jit': JITParticle}
 
 
-def set_globcurrent_fieldset(filename=None, indices=None, full_load=False):
+def set_globcurrent_fieldset(filename=None, indices=None, full_load=False, from_ds=False):
```

Comment:
I believe an explicit `use_xarray` would probably be better thatn `from_ds` here.